prow/jobs/jobs_config.yaml

```diff
@@ -45,4 +45,10 @@ soak_test_on_release_service_names: []
 code_gen_presubmit_services:
 - dynamodb
 - ecr
+- eks
```

Comment:
The EKS tests take >40 minutes to run. I'd be really hesitant to add this as it would block any fast changes to code-generator. Are you trying to validate a specific functionality in every code gen PR? What necessitated these changes?